src/mmw/js/src/data_catalog/models.js

```diff
@@ -24,6 +24,8 @@ var Catalog = Backbone.Model.extend({
         active: false,
         results: null, // Results collection
         resultCount: 0,
+        has_filters: false, // If local filters apply
+        options: [],        // Array of String Options
```

Comment:
Instead of adding an `options` attribute, consider subclassing the Catalog model for WDC and adding a `gridded` attribute. Historically, array attributes have [caused problems](https://stevenschobert.com/2013/01/03/arrays-and-objects-in-backbone-models) with Backbone models. I feel like the `addOption` and the `removeOptions` are somewhat duplicative of built-in Backbone methods. Also, none of the other catalogs have options yet, I'm not sure that this approach is necessary yet.